framework/rbac/BaseManager.php

```diff
@@ -115,6 +115,11 @@ public function createPermission($name)
     public function add($object)
     {
         if ($object instanceof Item) {
+            if($object->ruleName && $this->getRule($object->ruleName) === null){
```

Comment:
Could you add spaces before and after parentheses?

`if ($object->ruleName && $this->getRule($object->ruleName) === null) {`